config/config.json

```diff
@@ -77,7 +78,14 @@
       "repo": "OriginTrail/ot-node",
       "branch": "develop"
     },
-    "dataSetStorage": "data_set_storage"
+    "dataSetStorage": "data_set_storage",
+    "dc_holding_time_in_minutes": 60,
+    "dc_token_amount_per_holder": 1,
+    "dc_litigation_interval_in_minutes": 10,
+    "dh_max_holding_time_in_minutes": 100,
+    "dh_min_token_price": 1,
+    "dh_min_litigation_interval_in_minutes": 1,
+    "deposit_on_demand": true
```

Comment:
Remove old unused params for trading.

Reply:
Old properties are still used in some parts of the code, so we need to wait for that to disappear.